var/spack/repos/builtin/packages/gptune/package.py

```diff
@@ -2,11 +2,18 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
-
+import os
 
 from spack.package import *
 
 
+def terminate_bash_failures():
+    """Ensure bash scripts fail as soon as a command within fails."""
+    for f in os.listdir("."):
+        if f.endswith(".sh"):
+            filter_file(r"#!/bin/bash", r"#!/bin/bash" + "\nset -e", f)
```

Comment:
This would probably be easier to read if it accepts a directory as an argument instead of using the `cwd`.